File: app/src/main/java/com/example/android/popularmovies/displayUtilities/MoviesImageView.java
package com.example.android.popularmovies.displayUtilities;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.ImageView;



public class MoviesImageView extends ImageView {
    public MoviesImageView(Context context) {
        super(context);
    }

    public MoviesImageView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public MoviesImageView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, heightMeasureSpec);
        setMeasuredDimension(getMeasuredWidth(), getMeasuredHeight());
    }
}

File: app/src/main/java/com/example/android/popularmovies/displayUtilities/MoviesImageView.kt
package com.example.android.popularmovies.displayUtilities

import android.content.Context
import android.util.AttributeSet
import android.widget.ImageView

class MoviesImageView : ImageView {
  constructor(context: Context) : super(context) {}

  constructor(
    context: Context,
    attrs: AttributeSet
  ) : super(context, attrs) {
  }

  constructor(
    context: Context,
    attrs: AttributeSet,
    defStyleAttr: Int
  ) : super(context, attrs, defStyleAttr) {
  }

  override fun onMeasure(
    widthMeasureSpec: Int,
    heightMeasureSpec: Int
  ) {
    super.onMeasure(widthMeasureSpec, heightMeasureSpec)
    setMeasuredDimension(measuredWidth, measuredHeight)
  }
}
